docs/debug.md

```diff
@@ -0,0 +1,78 @@
+# How to debug recipes
+
+The Special Resource Operatore (SRO) can read the states either from the local,
+http or a ConfigMap resource.
+
+To create the states in the template directory as a CM we can run the
+following:
+
+```bash
+VERSION=0.0.1 REPO=example SPECIALRESOURCE=multi-build make assets
+```
+
+This command will create a CM with all the states and SRO will use them in the
```

Comment:
is this only for development or can be used in production also?
as the partner may not want to push the is assets into the SRO repo